docs/src/app/components/pages/components/table.jsx

```diff
@@ -332,7 +339,8 @@ export default class TablePage extends React.Component {
             type: 'function(rowNumber, columnId)',
             header: 'optional',
             desc: 'Called when a row cell is clicked. rowNumber is the row number and columnId is the column number ' +
-              'or the column key.',
```

Comment:
Pretty sure it's always just the column number.
The args I'm getting are `Event, lineNumber, colNumber`.
Should I update this ?